app/src/main/java/org/wikipedia/language/LanguageUtil.java

```diff
@@ -141,5 +143,19 @@ private static boolean isTraditionalChinesePredominantInCountry(@Nullable String
         return TRADITIONAL_CHINESE_COUNTRY_CODES.contains(country);
     }
 
+    @NonNull
+    public static String getFirstSelectedChineseVariant() {
+        if (FIRST_SELECTED_ZH_LANG_CODE == null) {
```

Comment:
What if the user changes the selected Chinese variant while the app is running?
Is this really so expensive that it needs to be cached?

Reply:
You're correct! and done.